packages/st2/debian/st2api-generator

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+NORMAL_DIR=${1:-/tmp}
+EARLY_DIR=${2:-/tmp}
+LATE_DIR=${3:-/tmp}
+
+ST2SVC="st2api"
+DEFAULT_IP="127.0.0.1"
+DEFFALT_PORT="9101"
```

Comment:
Typo DEFFALT_PORT
Should be DEFAULT_PORT

Reply:
Good catch, thanks for pointing that out!